### PR TITLE
=vertx deploy vertical target instances to match the CPU cores.

### PR DESCRIPTION
**Pull request by He-Pin**
**Summary of changes**
Changes introduced in this pull request:
- 
Set the target instances of vertical to match the CPU cores.

### CODE REVIEW

**Comment by He-Pin** [comment]
@gcnyin Ping , would you like to take a look?

**Comment by He-Pin** [comment]
/**
   * The default initial settings max concurrent stream for an HTTP/2 server = 100
   */
  public static final long DEFAULT_INITIAL_SETTINGS_MAX_CONCURRENT_STREAMS = 100;

Which can be ignore for now.

**Comment by gcnyin** [comment]
> @gcnyin Ping , would you like to take a look?

lgtm

File: java_vertx_grpc_bench/src/main/java/vertx/Main.java [hunk 1]
```diff
@@ -14,9 +15,11 @@
 public class Main {
     public static void main(String[] args) {
         Vertx vertx = Vertx.vertx();
-        vertx.nettyEventLoopGroup().forEach(u -> {
-            vertx.deployVerticle(new ServerVerticle());
-        });
+        DeploymentOptions deploymentOptions = new DeploymentOptions();
+        int cores = Runtime.getRuntime().availableProcessors();
```
Comment (LesnyRumcajs) [inline java_vertx_grpc_bench/src/main/java/vertx/Main.java:19]: Will this obtain a correct value of available processors? That is to say, in a Dockerized environment, will it grab the host's available parallelism or the container's? In my experience, it's usually the latter; this is the reason why the `GRPC_SERVER_CPUS` environmental variable is exposed to help the server properly determine how much is available.

Comment (He-Pin) [reply java_vertx_grpc_bench/src/main/java/vertx/Main.java:19]: IIRC currect the later JVM will works after JDK 11. I will update.

**Comment by He-Pin** [comment]
@gcnyin @LesnyRumcajs I have updated the pr and address issues.

**Review by LesnyRumcajs — APPROVED** [review]
Thank you!

**Pull request merged**